inst/shiny/global.R

```diff
@@ -30,7 +30,7 @@ var_meta <- c("SURVEY_NAM", "YT_REGION", "SURVEY_YEA", "SURVEY_ID",
               "PROJECT_ID", "CENSUS_ID",
               "In1Out0", "SU_STRATUM", "ALL_STRATA",
               "IDLATDEG", "IDLATMIN", "IDLONDEG", "IDLONMIN", "CENTRLAT", "CENTRLON",
-              "REGION", "GMU", "GMU2", "USE_SCALE", "SRC_SCALE", "SRC_NOTES",
+              "REGION", "MMU_ID", "GMU", "GMU2", "USE_SCALE", "SRC_SCALE", "SRC_NOTES",
```

Comment:
I've added MMU_ID to the list of meta data variables. This means that it is no long available to add as a predictor, which I think is correct...?

Reply:
Yes, that is correct AFAIK.